wvlet-lang/src/main/scala/wvlet/lang/compiler/parser/SqlToken.scala

```diff
@@ -260,6 +260,7 @@ object SqlToken:
     SqlToken.TRUE,
     SqlToken.FALSE,
     SqlToken.CASE,
+    SqlToken.IF,
```

Comment:
Do we really need to add IF as a special token here? Can't we just use a function name?